lib/tasks/price_policies.rake

```diff
@@ -0,0 +1,19 @@
+namespace :price_policies do
+
+  desc "Duplicate the newest price policies for a facility's services to the current fiscal year"
+  task :duplicate_to_current_fiscal_year, [:url_name] => :environment do |_t, args|
+    facility = Facility.find_by!(url_name: args[:url_name])
+
+    facility.services.active.each do |service|
+      service.price_policies.newest.each do |price_policy|
+        next unless price_policy.expired? # Don't duplicate a current policy
```

Comment:
The comment should maybe be `Don't duplicate an expired policy` here? 😄 Or am I missing something?

Reply:
The particular use case here is we're copying expired price policies into the current fiscal year. If the current fiscal year already has price policies, we don't want to copy them.

This is a fairly specific case, but figured putting it in the open branch would save us 15 minutes if we need to do again.

Reply:
Yep yep, makes sense—my brain parsed that `unless expired` backwards 🤦‍♂️